test/ui.test.js

```diff
@@ -217,7 +217,7 @@ describe('Component mounting test', () => {
           plugins: [ZapStore()],
         },
       })
-      expect(wrapper.html().length).toBeGreaterThan(100)
+      expect(wrapper.html().length).toBeGreaterThan(50)
```

Comment:
same what is this count?

Reply:
same here , since the extensions dialog is removed the components html string length will be shorter